FirebaseFunctions/Sources/Functions.swift

```diff
@@ -556,6 +655,58 @@ enum FunctionsConstants {
     return data
   }
 
+  @available(iOS 13, macCatalyst 13, macOS 10.15, tvOS 13, watchOS 7, *)
+  private func processResponseDataForStreamableContent(from data: Data?,
+                                                       error: Error?) throws
+    -> AsyncThrowingStream<
+      HTTPSCallableResult,
+      Error
+    > {
+    return AsyncThrowingStream { continuation in
+      Task {
+        var resultArray = [String]()
+        do {
+          if let error = error {
+            throw error
+          }
+
+          guard let data = data else {
+            throw NSError(domain: FunctionsErrorDomain.description, code: -1, userInfo: nil)
+          }
+
+          if let dataChunk = String(data: data, encoding: .utf8) {
+            // We remove the "data :" field so it can be safely parsed to Json.
+            let dataChunkToJson = dataChunk.split(separator: "\n").map {
+              String($0.dropFirst(6))
+            }
```

Comment:
nit: comment typo and I recommend following what the Firebase vertex SDK does and trim only the first 5 characters: https://github.com/firebase/firebase-ios-sdk/blob/8d75fefa3e2bff243e2588aa2f598a7ef649d5fd/FirebaseVertexAI/Sources/GenerativeAIService.swift#L144-L146

```suggestion
            // We remove the "data:" field so it can be safely parsed to Json.
            let dataChunkToJson = dataChunk.split(separator: "\n").map {
              String($0.dropFirst(5))
            }
```

Reply:
The JSON decoding should handle the leading space, and if the leading space isn't there, it won't be trimmed if we drop the first 5 characters.

Reply:
Thanks Nick, DONE.